grails-app/services/au/org/ala/profile/hub/ImageService.groovy

```diff
@@ -288,11 +288,34 @@ class ImageService {
         Map opus = model.opus
         String minusQuery = ""
         ImageOption opusDefaultOption = ImageOption.byName(opus.approvedImageOption, ImageOption.INCLUDE)
+        Integer numberOfIncludedLocalImages = 0;
+
+        switch (opusDefaultOption) {
+            case ImageOption.INCLUDE:
+                List excluded = profile?.imageSettings.findAll { isExcluded(opusDefaultOption, it?.displayOption?.toString()) }*.imageId
+                List privateImages = profile?.imageSettings.findAll {profile?.privateImages?.imageId.contains(it?.imageId?.toString())}
```

Comment:
this code is for finding all private images from profile imageSettings with same image Id from profile privateImages because there is no private info in imageSettings